unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/WorldState.cs

```diff
@@ -128,5 +130,43 @@ public HashSet<Vector2Int> GetAllLoadedScenesCoords()
 
             return allLoadedParcelCoords;
         }
+
+        public List<GlobalScene> GetActivePortableExperienceScenes()
+        {
+            List<GlobalScene> activePortableExperienceScenes = new List<GlobalScene>();
+
+            foreach (var globalSceneId in globalSceneIds)
+            {
+                if (loadedScenes.ContainsKey(globalSceneId))
+                {
+                    GlobalScene peScene = loadedScenes[globalSceneId] as GlobalScene;
+                    if (peScene.isPortableExperience)
+                    {
+                        activePortableExperienceScenes.Add(peScene);
+                    }
+                }
```

Comment:
I think it might be better to use `TryGetValue` if we are using it value 
```suggestion
                if (loadedScenes.TryGetValue(globalSceneId, out ParcelScene scene))
                {
                    GlobalScene peScene = scene as GlobalScene;
                    if (peScene.isPortableExperience)
                    {
                        activePortableExperienceScenes.Add(peScene);
                    }
                }
```

Reply:
I agree :) Done!